static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -44,6 +44,8 @@ type PageTargeting = {
     urlkw: string,
 };
 
+type PageTargettingLoose = { [key: string]: mixed };
+
```

Comment:
We extracted this `type` as it's being used in various places in the code.

In a further PR, we should review the `PageTargetting` type and make it handle all cases.